src/rf/rf/urls.py

```diff
@@ -19,8 +20,11 @@
 urlpatterns = patterns(
     '',
     url(r'^', include(apps.home.urls)),
+    url(r'^', include(apps.core.urls)),
```

Comment:
We usually keep URLs associated with an app inside that app, which seems sensible to me. Why do you prefer putting all the URLs in the home app?